src/widgets/PieChart.js

```diff
@@ -10,7 +10,8 @@ import { VictoryLabel, VictoryPie } from 'victory-native';
 
 import { APP_FONT_FAMILY, GREY } from '../globalStyles';
 
-export const PieChart = ({ width, height, data }) => {
+export const PieChart = ({ width, height, json }) => {
```

Comment:
hmm.. you seem to have renamed a bunch of these props to `json` (I'm not sure why), but you are still passing as `data` here: https://github.com/openmsupply/mobile/blob/f8b25f317a6b1d6e3a000c9d52f8e715b01ea817/src/widgets/ReportChart.js#L31-L38

I would just revert back to `data`?

Reply:
My mistake! Yes, I guess that's the best option, since we decided to keep it like `data`, otherwise I need to rename to `json` all the rest but not the best option.
Thanks btw😅

Reply:
Hopefully this is well done here bf1b7bd 😄